CONTRIBUTING.md

```diff
@@ -66,14 +64,20 @@ export USERNAME={userName}
 export PASSWORD={password}
 ```
 
-After setting up the environment variables, run this script to copy the configuration into the JSON configuration files:
+For Windows, please set the following environment variables - ISSUER, CLIENT_ID, USER_NAME, PASSWORD
```

Comment:
This might read a bit better as a list:
#

For Windows, please set the following environment variables:
- ISSUER
- CLIENT_ID
- USER_NAME
- PASSWORD